lib/scratch-buffers.c

```diff
@@ -316,3 +318,15 @@ scratch_buffers_global_deinit(void)
   stats_unregister_counter(&sc_key, SC_TYPE_QUEUED, &stats_scratch_buffers_bytes);
   stats_unlock();
 }
+
+void
+scratch_buffers_global_init(void)
+{
+  register_application_hook(AH_RUNNING, (ApplicationHookFunc) scratch_buffers_register_stats, NULL);
```

Comment:
With this we will be able to increase stats_level of counters whose level is currently 0 because of technical restrains. +1
Would you agree to increase those counters stats_level when this PR is merged?

Reply:
I am not sure which ones you mean, but if you mean scratch buffers specifically, I would leave them at zero, as they are a very good indicators of epic failure, and I don't think they have that much of an overhead. Don't forget that scratch buffers are not calculated into memory usage and those are also at zero.